docs/solana/migration/hotspot-maker.mdx

```diff
@@ -317,10 +317,69 @@ like to use the library, the setup guide can be found here
 
 ---
 
-## Data Credits and Sol for Onboarding
-
-<!-- TODO: We need to create an easy way for makers to get sol and data credits into their accounts -->
-
-```
-
-```
+## Data Credits and SOL for Onboarding
+
+:::info
+
+Maker Wallets will need both `SOL` and `DC` for transactions post Migration.
```

Comment:
Lets clarify that SOL is for the Solana transaction and the DC is for the onboarding fees.